Samples/Objective-C/nhubsample-refresh/nhubsample-refresh.xcodeproj/project.pbxproj

```diff
@@ -359,6 +386,7 @@
 				);
 				PRODUCT_BUNDLE_IDENTIFIER = "com.example.nhubsample-refresh";
 				PRODUCT_NAME = "$(TARGET_NAME)";
+				SUPPORTS_MACCATALYST = YES;
 				TARGETED_DEVICE_FAMILY = "1,2";
```

Comment:
Additional settings like this are checked by `VerifyNoBuildSettings.swift` to force move everything like that to config files.  Probably something goes wrong in this check if it compiles without error.

Reply:
Right, I need to move all things from the sample apps both to their own support files